css/style.css

```diff
@@ -231,14 +230,128 @@ width: 600px;
     margin-bottom: 20px;
 }
 
-.rent__text + .rent__text {
+.rent__text+.rent__text {
     margin-bottom: 20px;
 }
 
-.rent__item{
+.rent__item {
     border: 1px solid #332208;
     width: 250px;
- padding: 0 30px 10px;
+    padding: 0 35px 10px;
     text-align: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+
+
+}
+
+.rent__item-num {
+    font-family: "Fira Sans", sans-serif;
+    font-weight: 500;
+    font-size: 96px;
+    line-height: 115px;
+    opacity: 0.05;
+}
+
+.rent__item-title {
+    font-family: "Fira Sans", sans-serif;
+    font-size: 18px;
+    line-height: 22px;
+    margin-bottom: auto;
+}
+
+.benefits {
+    padding: 100px 0;
+}
+
+.benefits__title {
+    font-family: "Fira Sans", sans-serif;
+    font-size: 24px;
+    line-height: 29px;
+    margin-bottom: 50px;
+}
+
+.benefits__items {
+    display: flex;
+    justify-content: space-between;
 }
 
+.benefits__list {
+    width: 300px;
+    list-style: none;
+}
+
+.benefits__content {
+    width: 300px;
+    text-align: right;
+}
+
+.benefits__list-item {
+    margin-bottom: 27px;
+    padding-right: 26px;
+    position: relative;
+    text-align: right;
+}
+
+.benefits__list-item::after {
+    content: '';
+    position: absolute;
+    width: 16px;
+    height: 16px;
+    border-radius: 50%;
+    background-color: #332208;
+    right: 0;
+    top: 3px;
+}
+
+
+.benefits__list-item:nth-child(1),
+.benefits__list-item:nth-child(7) {
+    margin-right: 10px;
+}
+
+.benefits__list-item:nth-child(2),
+.benefits__list-item:nth-child(6) {
+    margin-right: 54px;
+}
+
+.benefits__list-item:nth-child(3),
+.benefits__list-item:nth-child(5) {
+    margin-right: 100px;
+}
+
+
+.benefits__list-item:nth-child(4) {
+    margin-right: 147px;
+}
+
+.benefits__text {
+    margin-bottom: 30px;
+    text-align: right;
+}
+
+.benefits__text+.benefits__text {
+    width: 267px;
+    margin-left: auto;
+}
+
+.video__popup {
+    background-color: #C5AC2D;
+    text-transform: uppercase;
+    color: #fefefe;
+    padding: 6px 34px 6px 64px;
+    text-align: center;
+    display: inline-block;
+    position: relative;
+}
+
+.video__popup::before {
+    content: '';
+    position: absolute;
+    width: 25px;
+    height: 25px;
+    background-image: url('../img/Group.png');

```

Comment:
use srcset and 2x image for background-image 
![image](https://github.com/Andyone89/carsharing/assets/39525140/a28fe117-2393-4c8a-964e-7e6425509b5c)